pkg/controller/nodenetworkstateconfiguration/nodenetworkstateconfiguration_controller.go

```diff
@@ -139,9 +142,50 @@ func (r *ReconcileNodeNetworkStateConfiguration) Reconcile(request reconcile.Req
 
 	nmstateOutput, err := nmstate.ApplyDesiredState(instance)
 	if err != nil {
-		return reconcile.Result{}, fmt.Errorf("error reconciling nodenetworkstate configuration at desired state apply: %v", err)
+		errmsg := fmt.Errorf("error reconciling nodenetworkstate at desired state apply: %v", err)
+		conditions.SetCondition(
+			instance,
+			nmstatev1alpha1.NodeNetworkStateConditionFailing,
+			corev1.ConditionTrue,
+			"Failed",
+			errmsg.Error(),
+		)
+		conditions.SetCondition(
+			instance,
+			nmstatev1alpha1.NodeNetworkStateConditionAvailable,
+			corev1.ConditionFalse,
+			"Success",
+			"successfully reconciled NodeNetworkState",
```

Comment:
Available is set to false. Messages should reflect that

Reply:
Should it be same as messages for Failed condition? @qinqon @phoracek

Reply:
It should all makes sense for Available=false. The same for the others.